pkg/dom0-ztools/rootfs/etc/init.d/000-mod-params

```diff
@@ -34,4 +34,6 @@ zfs_set_default_parameters() {
     set_module_parameter zfs zfs_smoothing_write 5
 }
 
+# Load ZFS and set parameters
+modprobe zfs
```

Comment:
Now that the ZFS module is loadable again, I'm wondering if doesn't make sense to pass all parameters during the loading instead of use /sys/module, that was set because module was built-in....

Reply:
So I did an attempt but it is not that simple with our container model. The description is here: https://github.com/lf-edge/eve/pull/3107#issuecomment-1479206338  I revoke the patch and leave as it is supposed to be - just a revert in order to fix the licensing problem. I assume we need to revisit this and fix how we call modprobe and make it proper way.